constants/checklist-data.ts

```diff
@@ -0,0 +1,362 @@
+import { CopyHolder } from '../types/language'
+
+import { Accordian } from '../types/accordian'
+export const checklistData: CopyHolder = {
+  english: {
+    title: '72 Hour Checklist',
+    description:
+      'Welcome back! We are wishing you a successful transition back into Santa Barbara County. The first 72 hours can be an important part of your transition. This guide will help you have a better understanding of some of the resources that you might need. We hope that these resources will be able to help you improve your quality of life.',
+  },
+
+  spanish: {
+    title: 'Lista de verificación de 72 horas',
+    description:
+      '¡Bienvenido de nuevo! Le deseamos una exitosa transición de regreso al condado de Santa Bárbara. Las primeras 72 horas pueden ser una parte importante de su transición. Esta guía le ayudará a comprender mejor algunos de los recursos que podría necesitar. Esperamos que estos recursos puedan ayudarlo a mejorar su calidad de vida.',
+  },
+}
+
+export const probationAccordian: Accordian = {
+  english: {
+    title: 'Report To Your Probation or Parole Officer',
+    description:
+      'When first releasing it is recommended to check in with your probation officer withing 24 hours, or as ordered by the Court, or as instructed by an officer. Please reach out to Probation or Parole in the resource below for any additional questions that you may have',
+    actionClick: 'blank',
+    itemOne: 'Santa Barbara Probation Agency',
+    itemTwo: 'Website:',
+    itemTwoUrl: 'www.sbprobation.org',
```

Comment:
because there is no `https://` prefix, this will attempt to find the `www.sbprobation.org` in our pages folder. this will cause a 404 error

Reply:
Fixed this one. Nice to know